collection.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+$db = new PDO('mysql:host=DB;dbname=collectiondb', 'root', 'password');
```

Comment:
the database connection should be in a function (this line and line 5) so you can reuse it. put it into a `functions.php` file